frontend-react/src/utils/misc.ts

```diff
@@ -46,6 +46,38 @@ export const checkJson = (
     }
 };
 
+export function includesSpecialChars(text: string): boolean {
+    return /[^a-z\d\s]/gi.test(text);
+}
+
```

Comment:
Probably some unit tests would be useful?

I think `includesSpecialChars("a\ttest")` returns false (e.g. someone copy/pastes a tab or linefeed character)

Also,
`encodeURI("a\ttest") !== "a\ttest"` returns true. But calling encodeURI maybe is slightly more expensive?


There also might need to be a `trim()` before checking? Not sure what level that might happen at.